### PR TITLE
chore: pointing to dotnet-core and backfilling from other dotnet repos

### PR DESCRIPTION
**Pull request by tanderson-ld**
The Dotnet SDKs are now living in dotnet-core, this PR is to fix the crawler.

### CODE REVIEW

File: metadata/launchdarkly_dotnet-core.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (cwaldren-ld) [inline metadata/launchdarkly_dotnet-core.json:1]: You can delete this file (or not, it will be fetched at crawl time and overwritten)

Comment (cwaldren-ld) [reply metadata/launchdarkly_dotnet-core.json:1]: Although, it looks like this metadata file has more info than the one checked into the repo: https://github.com/launchdarkly/dotnet-core/blob/main/.sdk_metadata.json

The repo's file should be updated to contain this content

Comment (tanderson-ld) [reply metadata/launchdarkly_dotnet-core.json:1]: There is a PR in the dotnet repo that will update it.  Is order of operations critical?

Comment (tanderson-ld) [reply metadata/launchdarkly_dotnet-core.json]: Deleted

**Review by cwaldren-ld — APPROVED** [review]

**Comment by cwaldren-ld** [comment]
You should probably merge the dotnet-core PR first, then this PR.

**Comment by cwaldren-ld** [comment]
ok looks like this could be merged now?

**Pull request merged**